pandas/conftest.py

```diff
@@ -995,6 +1004,14 @@ def _create_series(index):
 }
 
 
+@pytest.fixture(params=_series.keys())
+def series_with_differing_index(request):
```

Comment:
not sure if this is more idiomatic (might actually be less clear), but i think this could be

```
@pytest.fixture
def series_with_differing_index(indices):
     return _create_series(indices)
```